### DISCUSSION TITLE
在AX3600上安装报错Read-only file system

### DISCUSSION THREAD

**Discussion by chenxinw** [messense/aliyundrive-webdav#225] [Q&A] [answered]
root@XiaoQiang:/tmp# opkg install aliyundrive-webdav_1.1.0_aarch64_cortex-a53.ipk
Installing aliyundrive-webdav (1.1.0) to root...
Collected errors:
 * wfopen: //usr/lib/opkg/info/aliyundrive-webdav.conffiles: Read-only file system.
 * wfopen: //usr/lib/opkg/info/aliyundrive-webdav.control: Read-only file system.
 * wfopen: //usr/lib/opkg/info/aliyundrive-webdav.postinst: Read-only file system.
 * wfopen: //usr/lib/opkg/info/aliyundrive-webdav.prerm: Read-only file system.
 * wfopen: /usr/bin/aliyundrive-webdav: Read-only file system.
 * pkg_write_filelist: Failed to open //usr/lib/opkg/info/aliyundrive-webdav.list: Read-only file system.
 * opkg_install_pkg: Failed to extract data files for aliyundrive-webdav. Package debris may remain!
 * opkg_install_cmd: Cannot install package aliyundrive-webdav.

请问应该如何解决？

**Comment by messense** [comment, marked as answer]
刷 OpenWrt 而不是用小米默认的系统。
